uqcsbot/whatsdue.py

```diff
@@ -28,13 +27,13 @@
 
 def sort_by_date(item: AssessmentItem):
     """Provides a key to sort assessment dates by. If the date cannot be parsed, will put it with items occuring during exam block."""
```

Comment:
Occurring is misspelled here. Other spelling mistakes include 'seperated' and 'defualts'.